functions/Get-DbaLastBackup.ps1

```diff
@@ -142,22 +154,26 @@ function Get-DbaLastBackup {
                 }
 
                 $result = [PSCustomObject]@{
-                    ComputerName       = $server.ComputerName
-                    InstanceName       = $server.ServiceName
-                    SqlInstance        = $server.DomainInstanceName
-                    Database           = $db.Name
-                    RecoveryModel      = $db.RecoveryModel
-                    LastFullBackup     = [DbaDateTime]$LastFullBackup
-                    LastDiffBackup     = [DbaDateTime]$LastDiffBackup
-                    LastLogBackup      = [DbaDateTime]$LastIncrBackup
-                    SinceFull          = Get-DbaDateOrNull -TimeSpan $SinceFull_
-                    SinceDiff          = Get-DbaDateOrNull -TimeSpan $SinceDiff_
-                    SinceLog           = Get-DbaDateOrNull -TimeSpan $SinceLog_
-                    DatabaseCreated    = $db.createDate
-                    DaysSinceDbCreated = $daysSinceDbCreated
-                    Status             = $status
+                    ComputerName             = $server.ComputerName
+                    InstanceName             = $server.ServiceName
+                    SqlInstance              = $server.DomainInstanceName
+                    Database                 = $db.Name
+                    RecoveryModel            = $db.RecoveryModel
+                    LastFullBackup           = [DbaDateTime]$LastFullBackup
+                    LastDiffBackup           = [DbaDateTime]$LastDiffBackup
+                    LastLogBackup            = [DbaDateTime]$LastLogBackup
+                    SinceFull                = Get-DbaDateOrNull -TimeSpan $SinceFull_
+                    SinceDiff                = Get-DbaDateOrNull -TimeSpan $SinceDiff_
+                    SinceLog                 = Get-DbaDateOrNull -TimeSpan $SinceLog_
+                    LastFullBackupIsCopyOnly = $LastFullBackupIsCopyOnly
+                    LastDiffBackupIsCopyOnly = $LastDiffBackupIsCopyOnly # always false per https://docs.microsoft.com/en-us/sql/t-sql/statements/backup-transact-sql See comments above.
+                    LastLogBackupIsCopyOnly  = $LastLogBackupIsCopyOnly
+                    DatabaseCreated          = $db.createDate
+                    DaysSinceDbCreated       = $daysSinceDbCreated
+                    Status                   = $status
                 }
-                Select-DefaultView -InputObject $result -Property ComputerName, InstanceName, SqlInstance, Database, LastFullBackup, LastDiffBackup, LastLogBackup
+
+                Select-DefaultView -InputObject $result -Property ComputerName, InstanceName, SqlInstance, Database, LastFullBackup, LastDiffBackup, LastLogBackup, LastFullBackupIsCopyOnly, LastDiffBackupIsCopyOnly, LastLogBackupIsCopyOnly
```

Comment:
I see no reason to add the additional properties to the default output.

Reply:
no problem, I removed them from the default output.